### PR TITLE
Filters system

### PR DESCRIPTION
**Pull request by nicolas-brousse**

### CODE REVIEW

**Comment by B-Rass** [comment]
@nicolas-brousse maybe target `preprod` branch

File: config/application.rb [hunk 1]
```diff
@@ -39,6 +39,7 @@ class Application < Rails::Application
     # config.eager_load_paths << Rails.root.join("extras")
 
     config.i18n.default_locale = :fr
+    config.i18n.available_locales = %i[fr en]
```
Comment (B-Rass) [inline config/application.rb:42]: Do we want to add `en` now?

File: app/components/filter_component.rb [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+class FilterComponent < ApplicationComponent
+  renders_one :form, ->(&block) { FilterComponent::Form.new(&block) }
+
+  erb_template <<~ERB
+    <%= render CardComponent.new(type: :primary) do |card| %>
+      <% card.with_header do %>
+        <h3 class="panel-title"><%= t(".header.title") %><%= counter_tag %></h3>
+      <% end %>
+
+      <%#= filters_badge_tags %>
```
[the remark this answers is not in the document]
Comment (nicolas-brousse) [reply app/components/filter_component.rb:12]: It's commented so it's normal I guess

File: config/application.rb (hunk 1 above)
Comment (nicolas-brousse) [reply config/application.rb:42]: It's already add right now. This line will limit the locales files loaded. Without this, every locale files in gems are loaded.

File: app/components/filter_component.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (B-Rass) [reply app/components/filter_component.rb:12]: Indeed. Too subtle for my eyers I guess

File: config/application.rb (hunk 1 above)
Comment (B-Rass) [reply config/application.rb:42]: Ok perfect then 👌

**Review by B-Rass — APPROVED** [review]

**Pull request merged**